app/Models/Track.php

```diff
@@ -569,7 +580,7 @@ public function setTitleAttribute($value)
      */
     public function getFilesize($formatName)
     {
-        $trackFile = $this->trackFiles->where('format', $formatName)->first();
+        $trackFile = $this->trackFiles()->where('format', $formatName)->first();
```

Comment:
Why was this change necessary? It should be possible to query the Eloquent collection object returned by `trackFiles`.